src/actions/LoginActions.tsx

```diff
@@ -292,13 +292,13 @@ export function initializeAccount(
       const mergedDenominationSettings = {}
 
       for (const plugin of Object.keys(defaultDenominationSettings)) {
-        // @ts-expect-error
+        // @ts-expect-error - Dynamic object property assignment for denomination merging
```

Comment:
These comments are the wrong fix. The `mergedDenominationSettings` should be typed as `const mergedDenominationSettings: DenominationSettings = {}`. Then the error itself goes away.